src/v3/src/util/theme.ts

```diff
@@ -180,6 +180,11 @@ export const createThemeAndTokens = (
             paddingInlineEnd: mergedTokens.Spacing5,
             flexShrink: 0,
           },
+          message: {
+            '& .MuiLink-root': {
+
+            },
+          },
```

Comment:
Is this doing anything?

Reply:
I made this change as well as the `CssBaseline` one because of some complications with Odyssey's default styles. However, since you reviewed this they were finally able to publish the latest version of Odyssey to npm (they'd had issues for the past ~6 minor versions) and I just verified that the latest version fixes the issue that this line was solving.

The latest version _does_ have one issue with button text overflow, so once I push changes you'll see that I had to add one additional global style override to `MuiButton`

Reply:
If we need to wait for an Odyssey upgrade to remove this, can you add a comment indicating that (link a ticket as well)?

Reply:
Sure, I'll make a ticket and link it here